### DISCUSSION TITLE
Add a section with full file infos

### DISCUSSION THREAD

**Discussion by deltonio2** [jellyfin/jellyfin#5832] [General] [answered]
It would be awesome if there is the ability to show the full file details on the media page.

For exemple at the bottom, after the general information of a movie, show a section with the file informations: audio channels (2.0, 5.1, ...), audio language, audio and video codec, video resolution, size and extension of the file, HDR format, language subtitles, etc.

![Exemple](https://user-images.githubusercontent.com/8058756/115079707-e45cc200-9f01-11eb-8969-d3e3bce2b771.png)

**Comment by crobibero** [comment, marked as answer]
Feature requests belong at https://features.jellyfin.org/

**Reply by deltonio2** [reply]
@crobibero Sorry to be wrong, I create a feature request [here](https://features.jellyfin.org/posts/1138/add-a-section-with-full-file-infos).
